components/tikv_util/src/config.rs

```diff
@@ -522,6 +526,166 @@ impl<'de> Deserialize<'de> for ReadableDuration {
     }
 }
 
+#[derive(Clone, Debug, Copy, PartialEq)]
+pub struct ReadableOffsetTime(pub NaiveTime, pub FixedOffset);
+
+impl From<ReadableOffsetTime> for ConfigValue {
+    fn from(ot: ReadableOffsetTime) -> ConfigValue {
+        ConfigValue::OffsetTime((ot.0, ot.1))
+    }
+}
+
+impl From<ConfigValue> for ReadableOffsetTime {
+    fn from(c: ConfigValue) -> ReadableOffsetTime {
+        if let ConfigValue::OffsetTime(ot) = c {
+            ReadableOffsetTime(ot.0, ot.1)
+        } else {
+            panic!("expect: ConfigValue::OffsetTime, got: {:?}", c)
+        }
+    }
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize, Default)]
+pub struct ReadableSchedule(pub Vec<ReadableOffsetTime>);
```

Comment:
why is it vector?

Reply:
1. Because we want to be able to start this at a number of specified times. The examples this is based on does that.
2. Because with `Vec` i can derive Deserialize to toml. Since this is performance critical (O N is fine when N is small.), there is no need for it to be a Set.